region/maridia/inner-green/Oasis.json

```diff
@@ -312,6 +403,100 @@
       ],
       "devNote": "This only breaks the bomb blocks but doesnt get up to 6. However the requirements from here to 6 are less than 6->4."
     },
+    {
+      "link": [1, 6],
+      "name": "Spark to Break the Blocks",
+      "notable": false,
+      "entranceCondition": {
+        "comeInShinecharged": {
+          "framesRequired": 30
+        }
+      },
+      "requires": [
+        "h_canNavigateUnderwater",
+        {"shinespark": {"frames": 3}}
+      ],
+      "note": "Spark into the bomb blocks. The trick can fail with 50% probability due to collision oscillation.",
```

Comment:
Collision oscillation strats aren't good to have as the main way to do a room.  
There are some temp blue strats that are missing which would break the bomb blocks with Grav, HiJump, or springball and wouldn't use any health.